drawBot/context/baseContext.py

```diff
@@ -620,6 +645,10 @@ def intersectionPoints(self, other=None):
             contours += other._contoursForBooleanOperations()
         return booleanOperations.getIntersections(contours)
 
+    def strokePath(self, width, lineCap="round", lineJoin="round", miterLimit=10):
```

Comment:
Needs a good doc string.

Reply:
Could you please check if now it's good?

Reply:
maybe a different method name?

* `strokedPath`
* `expandPath`

strokePath is to ambiguous, it does not draw a stroked path...

Reply:
`expandPath` sounds better maybe, what do you think?

Reply:
for the docstring how about this:
```
        Returns an expanded stroke around the path with the given width.

        Optionally these options can be set:

        * `lineCap`: Possible values are `"butt"`, `"square"` or `"round"`
        * `lineJoin`: Possible values are `"bevel"`, `"miter"` or `"round"`
        * `mitterLimit`: The miter limit to use for `"miter"` lineJoin option
```

Reply:
something like this: `Returns a new bezier path with an expanded stroke around the bezier path with a given width.`

Reply:
sounds good. How about `expandStoke` for the method name?

Reply:
+1 for `expandStroke`